views/index.erb

```diff
@@ -0,0 +1,24 @@
+<link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
Looks like you spent some time formatting, nice one. One suggestion: have you looked into using CSS to style your HTMLs? It will make your HTMLs much easier to format and much easier to read